firebase-perf/src/main/java/com/google/firebase/perf/application/AppStateMonitor.java

```diff
@@ -315,54 +338,25 @@ public void onActivitySaveInstanceState(Activity activity, Bundle outState) {}
   @Override
   public void onActivityPaused(Activity activity) {}
 
-  /** Stops screen trace right after onPause because of b/210055697 */
-  @Override
-  public void onActivityPostPaused(@NonNull Activity activity) {
-    if (isScreenTraceSupported()) {
-      sendScreenTrace(activity);
-    }
-  }
-
   /**
-   * Send screen trace. If hardware acceleration is not enabled, all frame metrics will be zero and
-   * the trace will not be sent.
```

Comment:
Is this comment no longer true?

Reply:
This comment is true, but this is not relevant here. This has moved to FrameMetricsRecorder. So removed it.